### ISSUE TITLE
New order create issue

### ISSUE THREAD

**Issue by sainialpana** [magento/magento2#28664]
### Preconditions (*)
1) I am using Magento ver. 2.3.5.

### Steps to reproduce (*)
1) Login into backend.
2) Under sales > order > click on create new order
3)After click on add products, no data showing.

### Expected result (*)
1) List of my store products.

### Actual result (*)
1) No products shown.

**Comment by ajijshekh123** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ajijshekh123. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by sainialpana** [comment]
Hi @ajijshekh123 Have you checked the issue? Should i need to reinstall the M2? Please update me ASAP, its very urgent.

**Comment by ajijshekh123** [comment]
Sorry for Inconvenience @sainialpana, There are some issue on Magento Instance that's why I can't able to reverify it.
Let me try again.

**Comment by ajijshekh123** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ajijshekh123. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by sainialpana** [comment]
@ajijshekh123 Can you please check at my site backend?

**Comment by ajijshekh123** [comment]
Sure @sainialpana, Please share with me your site link and credentials on my slack account (Ajij Shekh).

**Comment by sainialpana** [comment]
> Sure @sainialpana, Please share with me your site link and credentials on my slack account (Ajij Shekh).

I don't have slack account. Please share your email id.

**Comment by ajijshekh123** [comment]
azizshaikh1987@gmail.com- This is my email id.

**Comment by sainialpana** [comment]
> azizshaikh1987@gmail.com- This is my email id.

Thanks

**Comment by sainialpana** [comment]
Sent email. please check.

**Comment by ajijshekh123** [comment]
Got Email.
Let me check and update you

**Comment by sainialpana** [comment]
> Got Email.
> Let me check and update you

Sure

**Comment by ajijshekh123** [comment]
Hello @sainialpana,
I have verified this issue on your Magento 2.3 Instance.
The issue is related to the Product list. First Fixed the Catalog issues then It will be automatically showing the products on Create order section.
Now, Catalog is not showing and It's displayed 'Content Security Policy' related error.
See attached the screenshot:

![Error](https://user-images.githubusercontent.com/24711761/84360159-4e705080-abe7-11ea-9e6b-c047ef37a547.PNG)

![Error2](https://user-images.githubusercontent.com/24711761/84360164-503a1400-abe7-11ea-9040-153f0644bc03.PNG)

I am QA Maintainer so I have just to verify the issue not to fix the issue. 

Thanks.

**Comment by sainialpana** [comment]
@ajijshekh123  Can you please ask technical team for this?

**Comment by ajijshekh123** [comment]
@sdzhepa - Can you please help to @sainialpana for this issue?

**Comment by sainialpana** [comment]
@ajijshekh123 

Are you working on this issue?
